actions-cleanup-openstack-vms/action.yaml

```
@@ -0,0 +1,85 @@
+name: 'Clean up OpenStack vms'
+description: 'this action is to clean up vms in openstack'
+
+
+inputs:
+  DELTA:
+    description: the default age of the VM
+    default: 6
+    required: false
+  CLOUD:
+    description: the cloud provider
+    required: false
+    default: 'openstack'
+  PASSWORD:
+    description: the password of the openstack client
```

```suggestion
    description: the password of the OpenStack client
```
to be fixed everywhere below